workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+
+# Create New Dashboard
+
+Create a new dashboard by selecting a data source to start your dashboard with:
+
+![](https://lh7-us.googleusercontent.com/awzI7eStfpqKWy78cOJdW-8pMyrVL6lrocPdQnmVlqrEtGh6QFjABrm-GoaCyG3URvGjdRnm_Ukzw4ZrbY6U-Li9QlY-eWKJ3dFzTzO_fb6IT48onLDEUZxaJ5EPDKJwWm6WLs_hzjmGJJTsCvq6AgQ)
+
+Select the metrics you would like to include as widgets in your dashboard:
+
+![](https://lh7-us.googleusercontent.com/MYBkwh9gWohgGbuF9CScQlUi5RU-0w0J8AVXTY3_MqEnidFnqTzGgXSLMy4iVcpV3_aRgrA3fCnxRZU53vGuL5Lp1RUL7ljVXmRdJfhbqptdDmcATY9LDiTDG715qWoX5ABfvTFuFVgmqArf6UC1i1I)
+
+Once you have added widgets to your dashboard, you will be taken directly to your dashboard:
+
+<Note> Data will appear momentarily after you have created your new dashboard </Note>
+
+![](https://lh7-us.googleusercontent.com/rg4LfJetovyC5UMrgE7H3aN3UUuK0K2kbbHp5LYgGHLs12M-48jW3OtN7RlPA0sJnTznOpcdtNFqghehlroWY7lhsw1eQsE1ZSKXSqf8e-EUNX99RzcCr24ScSxFiXhp27NvLokBGwV9PMuKWk8jwTY)
+
+# Create New Widget
+
+Navigate to the dashboard you would like to add a new widget to and select __Add Widget__ and __Create Widget__:
+
+![](https://lh7-us.googleusercontent.com/Qn2pnY24-2CuG7moVbGCi4UqO0aoIMdqNjsYaSipLHo3gfUOhD_z0vchbVyMNrN3zb6gPZeCAkkoJWmndzgqtcZVVFz9jzJYD_gkhP3qWNytupLefOGdMGLhd5Z9SjTvxqXbMmM1x6XYF0QzXK9LNn8)
+
+![](https://lh7-us.googleusercontent.com/Oe2a3IYXnQ4JTYXRbzMe9c7RZSHXFySFLJGFZCxWhJIFIvlNM2hOmGgQdCapl2h8Hk5-FhEWRwALQxZ1lJ2BGJtnvQnXA_r0TCqX5o3OeXwJnAqQrR1wkm77-uZLmL5V7QTiGpPr4Uhc6SRvjaO3Yjg)
+
+### Step 1: Select Chart Type
+
+Choose from one of the following types of charts for your widget:
+
+![](https://lh7-us.googleusercontent.com/LyMx_1CWIjqGEdzQYF_0XveQ3v-NvYJHLFA3N64iQfmfIx7KisKS0v2zx2VdMlB2caX9bAhYJUA9VULXsE8zGZHNH6XiBYbi0tjhlYTx8Uv5uBVKi80uDlceM9TUIGlZATlxwD6Pa1mJywRFU3XOYUQ)
+
+### Step 2: Create Query
+
+Create your query logic in the Metrics Section. Select the specific metric you would like to query, aggregate, and filter. Specify the interval at which data is ingested, create metric aliases to easily interpret what is displayed within the widget, and filter out unwanted data within the query.
+
+#### Group By
+
+The Group By field is the outer most aggregation made in your query. The way in which you aggregate a metric is contingent on the initial metric you have selected for your widget. Group your initial metric with resource attributes, metric attributes, or another metric based on the selected metric. The below example is grouping the `container.name` metric by all unique instances of the `container.name` metric:
+
+![](https://lh7-us.googleusercontent.com/lJ_3zm4C37Zgv8oWBRza4Z9-BbdJdlXD-lxbFq2qfGD579JQ9tNR_1MbC0zLLLW9ufQ-MQQa1_nwD4tg5PG-9QTpCDdsX5rE4UDIN-y9sILxyPlCHF8C8FcYv1od9VqK2ZACHS1NBAQFDk6yuOLgRzM)
+
+The following table displays all available methods for aggregating your chosen metrics:
+
+| __Aggregation Type__ | __Description__                                                |
```

Comment:
This is awesome!! Let's do this for widget type as well

Reply:
By widget type you mean the "Chart Type" to be displayed by the widget @Zamichel